modules/machineset-yaml-azure.adoc

```diff
@@ -73,17 +112,37 @@ spec:
           sshPublicKey: ""
           subnet: <infrastructureID>-<role>-subnet <1> <2>
           userDataSecret:
-            name: <role>-user-data <2>
+            name: worker-user-data <2>
           vmSize: Standard_D2s_v3
           vnet: <infrastructureID>-vnet <1>
```

Comment:
the vnet value does not seem to be a fixed value. Given infrastructureID is qeci-22538-spdj7, in some clusters it is like qeci-22538-vnet,in some clusters it is like qeci-22538-spdj7-worker-vnet. we can add a comment, you can obtain the vnet by running the following command:

```
  $  oc -n openshift-machine-api \
    -o jsonpath='{.spec.template.spec.providerSpec.value.vnet}{"\n"}' \
    get machineset/<infrastructureID>-worker-centralus1
```